src/timeMachine/utils/rawFluxDataTable.ts

```diff
@@ -14,6 +15,107 @@ export interface ParseFilesResult {
   maxColumnCount: number
 }
 
+export const parseFilesWithFromFlux = (
+  responses: string[]
+): ParseFilesResult => {
+  const chunks = parseChunks(responses.join('\n\n'))
+  const parsedChunks = chunks.map(c => fromFluxTableTransformer(c))
+  let data = []
+  parsedChunks.forEach((chunk, i) => {
+    data = data.concat(chunk)
+    if (i !== parsedChunks.length - 1) {
+      data.push([])
```

Comment:
an empty array signifies that the following data set should be treated as its own table